WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/management/InsightsManagementMapper.kt

```diff
@@ -47,17 +48,23 @@ private val ACTIVITY_INSIGHTS = mutableListOf(
 private val GENERAL_INSIGHTS = mutableListOf(
     ALL_TIME_STATS,
     MOST_POPULAR_DAY_AND_HOUR,
-    ANNUAL_SITE_STATS,
-    TODAY_STATS
+    ANNUAL_SITE_STATS
 )
 
 class InsightsManagementMapper @Inject constructor(
-    @Named(BG_THREAD) private val bgDispatcher: CoroutineDispatcher
+    @Named(BG_THREAD) private val bgDispatcher: CoroutineDispatcher,
+    private val trafficTabFeatureConfig: StatsTrafficTabFeatureConfig
 ) {
     suspend fun buildUIModel(addedTypes: Set<InsightType>, onClick: (InsightType) -> Unit) =
         withContext(bgDispatcher) {
             val insightListItems = mutableListOf<InsightListItem>()
             insightListItems += Header(R.string.stats_insights_management_general)
+            if (BuildConfig.IS_JETPACK_APP &&
```

Comment:
Since stats already work only on JP, we don't need `BuildConfig.IS_JETPACK_APP` here. I think we can remove all `BuildConfig.IS_JETPACK_APP` usages in this class.